.flowconfig

```diff
@@ -0,0 +1,7 @@
+[ignore]
```

Comment:
curious - is this file required?  If it's all defaults then I can keep going on my anti-dotfile tirade 😂